docs/site/Interceptors.md

```diff
@@ -56,7 +56,7 @@ method level interceptors. For example, the following code registers a global
 app
   .bind('caching-interceptor')
   .toProvider(CachingInterceptorProvider)
-  .apply(asGlobalInterceptor);
+  .apply(asGlobalInterceptor('caching'));
```

Comment:
is `caching` a binding key?

Reply:
No, it's the group name for the interceptor. We use group name to sort all global interceptors.

Reply:
do we want to mention it? Actually I was a bit confused when I was going through the documentation a while ago. :)

Reply:
Added more info.